Project21/Project21/comp.c

```diff
@@ -0,0 +1,41 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// complex.c
+// Functions performing on the complex numbers
+// Naveenkumar s
+// ------------------------------------------------------------------------------------------------
+
+#include"comp.h"
+#include<math.h>
+#include<stdio.h>
+
+Complex Op_Add (Complex c1, Complex c2) {
```

Comment:
Follow naming conventions
```suggestion
Complex OpAdd (Complex c1, Complex c2) {
```